azure-pipelines.yml

```diff
@@ -457,6 +507,7 @@ jobs:
     parameters:
       downloadPartialBuildBinFrom:
         - chip_earlgrey_cw310
+        - chip_earlgrey_cw310_hyperdebug
```

Comment:
Were you able to test if this download succeeds if the hyperdebug job did not run, or do we also need the `${{ if eq(variables['Build.SourceBranchName'], 'master') }}` construct used below on line 526?

Reply:
Not yet, but it'll get tested once I add the condition for running the job back ;)

You're probably right, though. I'll add the conditional inclusion of that job's results.